### PR TITLE
[LW-8192] chore: enable running on AArch64 (`conway-era`)

### PR DESCRIPTION
**Pull request by michalrus**
# Context

[LW-8192](https://input-output.atlassian.net/browse/LW-8192)

PR analogous to #1171 – look there for details.

# Proposed Solution

`yarn sanchonet:up` works 😌 

315598217-cdfa2749-e2b2-484d-914d-985388c50de6

[LW-8192]: https://input-output.atlassian.net/browse/LW-8192?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by mirceahasegan — APPROVED** [review]
Fantastic work, @michalrus ! 💪

File: compose/aarch64.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Overrides for AArch64 processors, until we get official images:
+
+services:
+  cardano-db-sync:
+    image: ghcr.io/input-output-hk/ogmios-tracker/cardano-db-sync:${CARDANO_DB_SYNC_VERSION:-sancho-4-0-0}
+  cardano-node:
+    image: ghcr.io/input-output-hk/ogmios-tracker/cardano-node:${CARDANO_NODE_VERSION:-8.8.0-pre}
+  ogmios:
+    image: ghcr.io/input-output-hk/ogmios-tracker/ogmios:v${OGMIOS_VERSION:-6.1.0}
```
Comment (michalrus) [inline compose/aarch64.yml:9]: We have to remember to update these versions here, if @mirceahasegan's PRs get merged first (#1181, #1184)

**Comment by michalrus** [comment]
@mirceahasegan but this one (I imagine, of more interest to you) has only 1 approval, anyone would like to add one more? :palms_up_together:

**Comment by mirceahasegan** [comment]
@michalrus I think we should first merge master -> conway-era, then rebase and fix conflicts.
If we merge this one first, there will be conflicts on the master->conway-era merge

**Comment by michalrus** [comment]
The diff was simplified thanks to @iccicci in #1209

**Review by iccicci — APPROVED** [review]

**Pull request merged**